### ISSUE TITLE
colors dont works

### ISSUE THREAD

**Issue by JustFoxx** [way2muchnoise/CurseForgeBadge#37]
i dont know but im trying change colors and when im trying to do https://cf.way2muchnoise.eu/short_adventure-of-fox-beggining-of-jurney(6441A4-E04E14).svg?badge_style=for_the_badge its just changing into black and idk what to do

**Comment by cech12** [comment]
I think two colors are not supported:

https://github.com/way2muchnoise/CurseForgeBadge-Unofficial/blob/master/CurseForgeBadge.py#L82

```
@app.route('/<style>_<project>_<prefix>_<suffix>(<l_colour>).svg')
@app.route('/<style>_<project>_<suffix>(<l_colour>-<r_colour>-<text_colour>-<shadow_colour>-<logo_colour>).svg')
@app.route('/<style>_<project>_<prefix>_<suffix>(<l_colour>-<r_colour>-<text_colour>-<shadow_colour>-<logo_colour>).svg')
```

You should try to define text/shadow/logo colors and you need a suffix (downloads?)
Something like that

https://cf.way2muchnoise.eu/short_564326_downloads(6441A4-E04E14-fff-fff-fff).svg?badge_style=for_the_badge

Hint: There seems to be an issue with "-" characters, so I changed it to your modpack id. :)

**Comment by way2muchnoise** [comment]
Yeah, the color stuff is not perfect 😅 and using the curseforge ID always solves a lot of issues as well
